spec/taxjar/api/request_spec.rb

```diff
@@ -168,6 +188,51 @@
       end
     end
 
+    context 'with a logger' do
+      let(:logger) { double('Logger 2') }
+
+      before do
+        stub_request(:get, "https://api.taxjar.com/api_path").
+          with(:headers => {'Authorization'=>'Bearer AK', 'Connection'=>'close',
+                            'Host'=>'api.taxjar.com',
+                            'User-Agent'=>"TaxjarRubyGem/#{Taxjar::Version.to_s}"}).
+          to_return(:status => 200, :body => '{"object": {"id": "3"}}',
+                    :headers => {content_type: 'application/json; charset=UTF-8'})
+
+        allow(logger).to receive(:info)
+        allow(logger).to receive(:debug)
+
+        described_class.logger = logger
+      end
+
+      after do
+        described_class.logger = nil
+      end
+
+      if HTTP::Options.available_features.key?(:logging)
```

Comment:
It appeared `logging` feature was introduced to HTTP.rb only in v4.0: https://github.com/httprb/http/blob/v4.0.0/lib/http/feature.rb

Unfortunately, since the gem is tested against Ruby <2.2 and HTTP.rb <4.0, to make test pass the example has to be conditional.